api/python/provisioner/inputs.py

```diff
@@ -859,9 +862,6 @@ class NetworkParams():
     secondary_data_netmask: str = ParamGroupInputBase._attr_ib(
         _param_group, descr="secondary node data interface netmask"
     )
-    secondary_data_network_iface: List = ParamGroupInputBase._attr_ib(
```

Comment:
Secondary can be deleted as each node has individual identity.
Plus the number of nodes are uncertain and should be queried from pillar data.